src/data/fintype/basic.lean

```diff
@@ -13,11 +11,58 @@ import data.array.lemmas
 import order.well_founded
 import group_theory.perm.basic
 
+/-!
+# Finite types
+
+This file defines a typeclass to state that a type is finite.
+
+## Main declarations
+
+* `fintype α`:  Typeclass saying that a type is finite. It takes as fields a `finset` and a proof
+  that all terms of type `α` are in it.
+* `finset.univ`: The finset of all elements of a fintype.
+* `fintype.card α`: Cardinality of a fintype. Also `univ.card`.
```

Comment:
```suggestion
* `fintype.card α`: Cardinality of a fintype. Equal to `finset.univ.card`.
```
Try to use fully-qualified names where possible so that the links behave.